plugins/hydra_tsp_launcher/hydra_plugins/hydra_tsp_launcher/tsp_launcher.py

```diff
@@ -0,0 +1,222 @@
+
+def look_for_task_spooler():
+    """
+    Task spooler is known as `tsp` on Ubuntu and `ts` on MacOS. 
+    This function checks for which one exists and returns the
+    valid command, or throws an error if neither is found.
+
+    Returns:
+        str: Either `ts` or `tsp` depending on which is valid.
+
+    Raises:
+        RuntimeError when TaskSpooler isn't found on the system.
+    """
+    commands = {'ts': False, 'tsp': False}
+
+    for c in commands:
+        try:
+            cmd_output = str(subprocess.check_output([c, '-h']))
+            if 'usage' in cmd_output:
+                commands[c] = True
+        except:
+            commands[c] = False
+    
+    if not any(commands.values()):
+        raise RuntimeError("Task spooler not found! Install it to use this launcher.")
+    
+    for c in commands:
+        if commands[c]:
+            return c
+
+def _get_job_status(tsp_prefix, job_id):
+    job_status = subprocess.check_output(
+        f"{tsp_prefix} -s {job_id}", shell=True)
+    return job_status.rstrip().decode('utf-8')
+
+def execute_job(
+    idx: int,
+    overrides: Sequence[str],
+    config_loader: ConfigLoader,
+    config: DictConfig,
+    task_function: TaskFunction,
+    singleton_state: Dict[Any, Any],
+    cmd_prefix: str,
+    tsp_prefix: str,
+) -> JobReturn:
+    """Calls `run_job` in parallel
+    """
+    setup_globals()
+    Singleton.set_state(singleton_state)
+
+    lst = " ".join(overrides)
+
+    sweep_config = config_loader.load_sweep_config(config, list(overrides))
+    with open_dict(sweep_config):
+        sweep_config.hydra.job.id = "{}_{}".format(sweep_config.hydra.job.name, idx)
+        sweep_config.hydra.job.num = idx
+    HydraConfig.instance().set_config(sweep_config)
+
+    def tsp_task_function(task_cfg):
+        working_dir = os.getcwd()
+        cmd = f"{cmd_prefix} {lst}"
+        log.info(f"\t#{idx} : {lst}")
+        cmd = f"cd {hydra.utils.get_original_cwd()} && {cmd} hydra.run.dir={working_dir}"
+        job_id = int(subprocess.check_output(cmd, shell=True).rstrip())
+        log.info(f"Submitted {idx} to TaskSpooler. View logs: {tsp_prefix} -t {job_id}")
+        return job_id
+
+    ret = run_job(
+        config=sweep_config,
+        task_function=tsp_task_function,
+        job_dir_key="hydra.sweep.dir",
+        job_subdir_key="hydra.sweep.subdir",
+    )
+    ret.id = ret.return_value
+
+    return ret
+
+class TaskSpoolerLauncherSearchPathPlugin(SearchPathPlugin):
+    """
+    This plugin is allowing configuration files provided by the ExampleLauncher plugin to be discovered
+    and used once the ExampleLauncher plugin is installed
+    """
+
+    def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
+        # Appends the search path for this plugin to the end of the search path
+        search_path.append(
+            "hydra-tsp-launcher",
+            "pkg://hydra_plugins.hydra_tsp_launcher.conf",
+        )
```

Comment:
You don't need the SearchPathPlugin because your plugin does not need to export config files.